index.html

```diff
@@ -8,14 +8,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"  />
     <link rel="stylesheet" href="style.css" type="text/css" />   
 </head>
+
 <body>
     <form>
         <h1>Stwórz swoją własną postać!!!</h1>
-        <label class="name-style" for="name">Imię:</label><br>
+        <label id="name-style" for="name">Imię:</label><br>
         <input type="text" class="your-name" name="y-name" value=""><br>
-        <label class="nick-style" for="nick">Nowy Nick:</label><br>
+        <label id="nick-style" for="nick">Nowy Nick:</label><br>
```

Comment:
In previous project I mentioned couple of times, that id shouldn't be used for styling. So - let's back to class. 
Here is a proper example (implement to all cases):
```
<label class="nick-style" for="nick">Nowy nick:</label><br>
<input type="text" id="nick" class="your-name" name="y-nick" value="">
```
So, id is assigned to type. So - label knows which element describes.